float64 torque
float64 duration
---
bool ok
